ui/lib/core/addon/templates/components/masked-input.hbs

```diff
@@ -1,35 +1,41 @@
-<div class="masked-input {{if shouldObscure "masked"}} {{if displayOnly "display-only"}} {{if allowCopy "allow-copy"}}"
+
+<div class="masked-input {{if displayOnly "display-only"}} {{if allowCopy "allow-copy"}}"
   data-test-masked-input data-test-field>
   {{#if displayOnly}}
-    <pre class="masked-value display-only is-word-break">{{displayValue}}</pre>
-  {{else if maskWhileTyping}}
+    <pre class="masked-value display-only is-word-break {{unless showValue "masked-font"}}">{{unless showValue (truncate value 20) value}}</pre>
+  {{else if inputField}}
     <input
       autocomplete="off"
       spellcheck="false"
-      type={{if isMasked "password" "text"}}
```

Comment:
previous work on LDAP bindpass secret. Basically a quick fix for this larger obscuring project. See PR [here](https://github.com/hashicorp/vault/pull/10087
).